composer.json

```
@@ -26,7 +26,7 @@
         "ext-openssl": "*",
         "ext-simplexml": "*",
         "ext-zip": "*",
-        "api-platform/core": "^2.7",
+        "api-platform/core": "^3.2.13",
```

```suggestion
        "api-platform/core": "^3.2",
```
This should be enough so we can update patch versions more easily if required